package.json

```diff
@@ -8,37 +8,44 @@
     "start": "next start",
     "start-build": "yarn build & yarn start",
     "clean": "rm -rf .next/ out/ node_modules/",
-    "cypress:start": "yarn run cypress open",
+    "cypress:start": "yarn cypress open",
     "cypress:start-build": "concurrently --names 'CYPRESS,SERVER' --prefix-colors 'yellow,blue' \"yarn cypress:start\" \"yarn start-build\"",
     "typedoc:build": "node_modules/.bin/typedoc --tsconfig .",
-    "typedoc:start": "npx serve docs"
+    "typedoc:start": "npx serve docs",
+    "jest:test": "jest --watchAll --verbose",
+    "test": "cypress run"
```

Comment:
so this is headless test huh? nice! We should probably run this for our workflow tests

Reply:
The headless test is already included under `.github/workflows`